README.md

```diff
@@ -58,16 +58,28 @@ Tested for python 3.5
 > To run an example with petsc (parallel), you need to run python with `mpirun`. For instance, to run a program with 2 proc
 > `mpirun -n 2 python3 myprog.py`
 
-## Install (without pip)
+Install 
 --------------------------------------
+You'll need:
+* python; 
+* python3-tk library;
```

Comment:
not sure about ` python3-tk` dependency. Could you remove it? and just say that python installation are required.
Ok for all other modifications.